app/js/app.js

```diff
@@ -0,0 +1,106 @@
+//responsive canvas
+$(document).ready(function(){
+  //get canvas and context
+  var c = $('#juice-canvas');
+  var ctx = c.get(0).getContext('2d');
+  var container = $(c).parent();
+  //when browser resizes, run function
+  $(window).resize(responsiveCanvas);
+  function responsiveCanvas(){
+    c.attr('width', $(container).width()); //max width
+    c.attr('height', $(container).height()); //max height
+  }
+  responsiveCanvas();
+});
+
+//mobile and desktop
+var c = document.getElementById('juice-canvas');
+var ctx = c.getContext('2d');
+var isDrawing = false;
+ctx.strokeStyle = '#333';
+ctx.lineWidth = 2;
+//touch
+c.addEventListener('touchstart', function(e) {
+  var rect = c.getBoundingClientRect();
+  var x = Math.round((e.clientX-rect.left) / (rect.right-rect.left) * c.width);
+  var y = Math.round((e.clientY-rect.top) / (rect.bottom-rect.top) * c.height);
+  var touch = e.touches[0];
+  var mouseEvent = new MouseEvent('mousedown', {
+    clientX: touch.clientX,
+    clientY: touch.clientY
+  });
+  c.dispatchEvent(mouseEvent);
+}, false);
+
+c.addEventListener('touchmove', function(e) {
+  var touch = e.touches[0];
+  var mouseEvent = new MouseEvent('mousemove', {
+    clientX: touch.clientX,
+    clientY: touch.clientY
+  });
+  c.dispatchEvent(mouseEvent);
```

Comment:
i like the approach you're taking here, but just mapping every touch even to a mouse event 👍